stan/math/prim/scal/fun/log_inv_logit_diff.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef STAN_MATH_PRIM_SCAL_FUN_LOG_INV_LOGIT_DIFF_HPP
+#define STAN_MATH_PRIM_SCAL_FUN_LOG_INV_LOGIT_DIFF_HPP
+
+#include <stan/math/prim/scal/fun/log1m_exp.hpp>
+#include <stan/math/prim/scal/fun/log1p_exp.hpp>
+#include <boost/math/tools/promotion.hpp>
+
+namespace stan {
+namespace math {
+
+/**
+ * Returns the natural logarithm of the difference of the
+ * inverse logits of the specified arguments.
+ *
+   \f[
+     \mbox{log_inv_logit_diff}(x,y) =
```

Comment:
This won't work in LaTeX without escaping `_` as `\_`.  

And `\mathrm{}` is better as it scales with its environments' size.  So that's `\mathrm{log\_inv\_logit\_diff}`.